src/frontend/apps/impress/src/features/docs/doc-header/components/DocVersionHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import { useTranslation } from 'react-i18next';
+
+import { Box, Text } from '@/components';
+import { HorizontalSeparator } from '@/components/separators/HorizontalSeparator';
+import { useCunninghamTheme } from '@/cunningham';
+import { Doc } from '@/features/docs/doc-management';
+
+interface DocHeaderProps {
+  doc: Doc;
```

Comment:
To avoid rerendering.

```suggestion
  title: string;
```